src/components/Book/Book.jsx

```diff
@@ -0,0 +1,31 @@
+import style from './Book.module.scss';
+import BookModal from '../BookModal/BookModal';
+import { useState } from 'react';
+
+const Book = ({book}) => {
+  const { volumeInfo } = book
+
+  const [show, setShow] = useState(false);
+
+  const handleShow = () => {
+    setShow(true);
+  }
+
+  const onClose = () => {
+    setShow(false);
+  }
+
+
+  return (
+    <>
```

Comment:
you could add a pure js function called for example cleanData that will take in api response (array of objects) and return an array of bojects with default values provided if somethig is missing